### PR TITLE
Fixes lowend functions - Changes PNG to default

### PR DESCRIPTION
**Pull request by locbones**
Program would error when saving to mod directory or any directory that included a . (period)
Re-did split argument to specify filename and extension only.

### CODE REVIEW

File: D2RModding-SpriteEdit/MainForm.cs [hunk 1]
```diff
@@ -742,5 +843,10 @@ private void onFrameCountChanged(object sneder, EventArgs e)
         {
             currentFrameCount = UInt32.Parse(numFramesTextBox.Text);
         }
+
+        private void imagePreview_Click(object sender, EventArgs e)
```
Comment (CollinHerber) [inline D2RModding-SpriteEdit/MainForm.cs]: Remove this?

File: D2RModding-SpriteEdit/MainForm.Designer.cs [hunk 2]
```diff
@@ -109,84 +111,91 @@ private void InitializeComponent()
...
             this.combineFramesToolStripMenuItem.Text = "Import Frames...";
             this.combineFramesToolStripMenuItem.Click += new System.EventHandler(this.combineFramesToolStripMenuItem_Click);
             // 
             // massTranslateToolStripMenuItem
             // 
             this.massTranslateToolStripMenuItem.Name = "massTranslateToolStripMenuItem";
-            this.massTranslateToolStripMenuItem.Size = new System.Drawing.Size(160, 22);
-            this.massTranslateToolStripMenuItem.Text = "Mass Translate...";
+            this.massTranslateToolStripMenuItem.Size = new System.Drawing.Size(180, 22);
+            this.massTranslateToolStripMenuItem.Text = "Bulk Import...";
```
Comment (CollinHerber) [inline D2RModding-SpriteEdit/MainForm.Designer.cs:156]: Nice! Thought about doing this myself haha.

File: D2RModding-SpriteEdit/MainForm.cs (hunk 1 above)
Comment (locbones) [reply D2RModding-SpriteEdit/MainForm.cs]: Removed - thanks

**Comment by CollinHerber** [comment]
Closing in favor of #2

**Pull request closed without merging**